tools/nl/embeddings/run.sh

```diff
@@ -14,18 +14,19 @@
 # limitations under the License.
 
 function help {
-  echo "Usage: -bfl <embeddings-size> # 'small' or 'medium'"
+  echo "Usage: -bflc <embeddings-size>"
   echo "$0 -b <embeddings-size> # 'small' or 'medium'. This option uses the base default sentence_transformer model."
   echo "$0 -f <embeddings-size> # 'small' or 'medium'. This option uses the finetuned model on PROD."
   echo "$0 -l <embeddings-size> <local_model_path> # 'small' or 'medium'. This option uses the locally stored model to build the embeddings."
+  echo "$0 -c <embeddings-size> <local_sheets_csv_filepath> <sheets_url> <worksheet_name> # This option creates custom embeddings (using the finetuned model in PROD)."
```

Comment:
Ok now i am a bit more confused :) Why provide the local path AND the sheets url+worksheet? They are one and the same thing, no? The local path is where the file from sheets_url+worksheet is downloaded to, no?

Reply:
sorry for my confusion! it looks like the sheets url+worksheet is just getting written to the local path, so I added an option to avoid overwriting the existing file that's checked in. but I guess it doesn't matter if it's overwritten (can choose not to push, or can rerun with small/medium), so I removed this flag